packages/legend-studio/src/stores/EditorStore.ts

```diff
@@ -1428,12 +1428,24 @@ export class EditorStore {
       .map((e) => buildElementOption(e) as PackageableElementOption<Service>);
   }
 
+  // may make sense to add to config
+  get TEMPORARY_supportNewServiceTestableFlow(): boolean {
```

Comment:
Yes, let's make this a config in `LegendStudioApplicationConfig`, also, please rename this to `TEMPORARY__supportNewServiceTestableFlow`, notice we use `__` separator in these kinds of names in the codebase.

Reply:
removing it. was taking a light approach initially, but decided to remove old editor.